app/pages/organisation/contextCreator.js

```diff
@@ -0,0 +1,24 @@
+import manifest from './manifest.json';
+
+export const getContext = ({ data }) => ({
+  ...manifest,
+  orgId: data.organisationId,
+  orgName: data.name,
+  odsCode: data.odsCode,
+  primaryRoleId: data.primaryRoleId,
+  address: data.address ? data.address.split(',') : [],
+  agreementSigned: data.isCatalogueAgreementSigned,
+  columnInfo: manifest.columnInfo ? manifest.columnInfo : [],
+  data: data && data.users ? data.users.map(row => [
+    { data: row.name || '', href: '#' || '#' },
+    { data: row.telephone || '' },
+    { data: row.email || '' },
+    {
+      tag: row.isDisabled ? {
+        dataTestId: `account-disabled-tag-${row.userId}`,
+        classes: 'bc-c-tag-outline nhsuk-u-font-size-16',
+        text: 'ACCOUNT DISABLED',
```

Comment:
I'm just bit concerned here that we are mixing in presentation data with the context creator. I personally think this should live in the template so we know the presentation element is there. What do you think?

Reply:
I did this here so that I could keep the table generic. If I did it in the table then any tag in this table would have these classes and text

Reply:
OK I'm with it now. I don't mind the text coming from the contextCreator. It's just the classes. It wouldn't be the first place I look if say we wanted to change these stylings.